lua/swamp/misc/sh_titles.lua

```diff
@@ -160,6 +160,69 @@ AddTitle("kleinertp", {
     {1, "Test Subject", 10000}
 }, "Be subjected to one of Dr. Isaac Kleiner's teleportation experiments", "s_kleinertp")
 
+AddTitle("buttonsearch", {
+    { 1, "Button Rookie", 0},
+    { 10, "Button Hunter", 0},
+    { 100, "Button Master", 0}
+}, "Find and press %s buttons.", "s_magicbutton")
+
+AddTitle("snowballhit", {
+    {100, "Frosty The Snowman"}
+}, "Hit %s active players with a snowball.", "s_snowballhit")
+
+AddTitle("drinkenergy", {
+    {100, "Boomer"},
+    {500, "Quake Was A Good Game"},
+    {1000, "Quake Champion"}
+}, "Drink your boomer juice %s times", "s_drinkenergy")
+
+AddTitle("bountykill", {
+    {10, "Bounty Hunter", 0},
+    {100, "The Cleaner", 0},
+    {1000, "Hitman", 0}
+}, "Collect %s bounties.", "s_bounties")
+
+AddTitle("headshotkill", {
+    {50, "Boom Headshot!", 2500},
+    {250, "American Sniper", 10000},
+    {1000, "Lee Harvey Oswald", 50000}
+}, "Kill %s non-afk players with headshots.", "s_headshotkill")
+
+AddTitle("kleinerkiller", {
+    {100, "Kleiner Killer", 5000},
+    {1000, "Anti Kleiner", 10000},
+    {10000, "Kleiner Exterminator", 25000}
+}, "Kill %s Kleiners.", "s_kleinerkiller")
+
+AddTitle("knifekill", {
+    {25, "Edgelord", 0},
+    {100, "Stab Enthusiast", 0},
+    {1000, "American Psycho", 0}
+}, "Kill %s non-afk players with the throatneck slitter.", "s_knifekill")
+
+AddTitle("theaterkill", {
+    {100, "Peacekeeper", 0}
+}, "Defend your theater %s amount of times.", "s_theaterkill")
+
+AddTitle("fistkill", {
+    {25, "Fightclub Member", 5000},
+    {100, "Chad", 10000},
+    {500, "Giga Chad", 25000},
+    {1000, "Billy's Disciple", 100000}
+}, "Get %s kills with the fists.", "s_fistkill")
+
+AddTitle("dodgeballkill", {
+    {25, "FatKid", 2500},
```

Comment:
should use titles like "Jock"

Reply:
I can't think of any besides Jock and Bully which I have slotted in.